src/styles/index.scss

```diff
@@ -2,2 +1,208 @@
+@import 'variables';
+@import 'stars';
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+// Header block
+
+.header {
+  width: 100%;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+
+  &__container {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-inline: 50px;
+  }
+
+  &__logo-link {
+    width: 40px;
+    height: 40px;
+  }
+}
+
+.nav {
+  height: 60px;
+  display: flex;
+
+  &__list {
+    padding: 0;
+    display: flex;
+    align-items: center;
+    list-style: none;
+  }
+
+  &__list-item .nav__item {
+    display: flex;
+    align-items: center;
+    text-align: center;
+    height: 60px;
+    margin-left: 20px;
+    font-size: 12px;
+    font-weight: 500;
+    color: #060b35;
+    text-transform: uppercase;
+    text-decoration: none;
+    transition: all 0.3s;
+  }
+
+  &__list-item:first-child .nav__item {
+    margin-left: 0;
+  }
+
+  &__list-item .nav__item:hover {
+    color: #00acdc;
+  }
+
+  &__list-item .is-active {
+    position: relative;
+    color: #00acdc;
+  }
+
+  .is-active::after {
+    content: '';
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    height: 4px;
+    border-radius: 8px;
+    background-color: #00acdc;
+  }
+}
+
+// Card block
+
+.main {
+  display: grid;
+  padding: 50px 40px;
+  grid-gap: 48px 46px;
+  grid-template-columns: 1fr;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
```

Comment:
Media queries should be placed at the end of the SCSS file to ensure they are not scattered throughout the file. This makes it easier to find and manage responsive styles.